src/common-principles.md

```diff
@@ -430,36 +430,41 @@ NIfTI header.
 
 ### Tabular files
 
-Tabular data MUST be saved as tab delimited values (`.tsv`) files, that is, CSV
-files where commas are replaced by tabs. Tabs MUST be true tab characters and
-MUST NOT be a series of space characters. Each TSV file MUST start with a header
-line listing the names of all columns (with the exception of
-[physiological and other continuous recordings](modality-specific-files/physiological-and-other-continuous-recordings.md)
-as well as [motion recording data](modality-specific-files/motion.md)).
+Tabular data MUST be saved as plain-text, tab-delimited values (`.tsv`) files,
+that is, CSV files where commas are replaced by tab characters.
+Tabs MUST be true tab characters and MUST NOT be a series of space characters.
+Tabular data containing large numbers of rows MAY be saved as
+[compressed tabular files (with extension `.tsv.gz`)](#compressed-tabular-files)
+as prescribed below.
+Each TSV file MUST start with a header line listing the names of all columns
+with the exception of [compressed tabular files](#compressed-tabular-files)
+where column names are defined in a sidecar metadata
+[JSON object](https://www.json.org/json-en.html) described below.
```

Comment:
Watch out - there are small consistency changes that are less apparent because of the use of semantic line breaks - hope that's fine for such a short excerpt.